airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1960,6 +1960,8 @@ components:
         email:
           type: string
           format: email
+        name:
```

Comment:
I think that we can also use the review to update the slug as well.

Reply:
Hi, thanks for pointing this out. I've made sure the slug is updated based on the new name. I assume we don't want (can't) let the user specifcy this, since the code we currenlty use anyway needs to "retry" until it's unique? Or would we want to let the API caller explicitly specify a slug and fail the request if that's already taken?

Could you also point me to a place the slug is used? It seems in URLs we're using the IDs of the workspace and not the slug?